eclipse-collections/pom.xml

```diff
@@ -42,6 +42,13 @@
             <version>${project.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.google.auto.service</groupId>
+            <artifactId>auto-service</artifactId>
+            <version>1.0-rc4</version>
+            <optional>true</optional>
+        </dependency>
```

Comment:
auto-service was a convenient way for me to register stuff with the ServiceLoader, but it doesn't carry its own weight. And it depends on Guava. The production implementation of this should just mess with META_INF directly, without using a framework.